package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "hyperswitch",
-  "version": "0.11.1",
+  "version": "0.11.2",
```

Comment:
I would disagree with @d00rman here - we are dropping support for swagger 2.0, so it does deserve a minor version bump

Reply:
As I wrote in my previous review, I think we should keep the compatibility with swagger v2 in this version bump and then in a second step remove support for it

Reply:
I think it would be too hard, to be honest. What are we gaining from maintaining compatibility and removing it later?

Reply:
Hm you are actually correct @Pchelolo, in this instance it is not trivial to maintain compatibility. So yeah, let's bump the minor version as it will not be compatible with the current hyperswitch version.

Reply:
Done.